multivariate-fine-mapping/mixture_prior.ipynb

```diff
@@ -437,21 +443,24 @@
     "parameter: ud_method = \"ed\"\n",
     "# A list of models where we only update the scales and not the matrices\n",
     "# A typical choice is to estimate scales only for canonical components\n",
-    "parameter: scale_only = []\n",
-    "\n",
+    "parameter: scale_only = \"canonical\"\n",
     "input: [f\"{cwd}/{name}.rds\"] + [f\"{cwd}/{name}.{m}.rds\" for m in mixture_components]\n",
     "output: f'{cwd}/{name}.{ud_method}{\"_unconstrained\" if len(scale_only) == 0 else \"\"}{(\".\" + os.path.basename(resid_cor)[:-4]) if resid_cor.is_file() else \"\"}.rds'\n",
     "task: trunk_workers = 1, walltime = '36h', trunk_size = 1, mem = '10G', cores = 4, tags = f'{_output:bn}'\n",
     "R: expand = \"${ }\", stderr = f\"{_output:n}.stderr\", stdout = f\"{_output:n}.stdout\",container = container\n",
+    "    library(stringr)\n",
     "    rds_files = c(${_input:r,})\n",
     "    dat = readRDS(rds_files[1])\n",
     "    U = list(XtX = dat$XtX)\n",
     "    U_scaled = list()\n",
-    "    for (f in rds_files[2:length(rds_files)]) {\n",
-    "        if (f %in% scale_only) {\n",
-    "          U_scaled = c(U_scaled, readRDS(f))\n",
+    "    mixture_components = c('${\"','\".join(mixture_components)}')\n",
```

Comment:
A shortcut to implement this would be `c(${paths(mixture_components):r,})` where `r` means raw str with quotes, `,` means join by comma. This is shorthand only available to `path` , `paths` and a few other SoS objects.